demos/python_demos/colorization/colorization_demo.py

```diff
@@ -0,0 +1,106 @@
+from openvino.inference_engine import IENetwork, IECore
```

Comment:
Shebang line and license header should be added.

Reply:
Name of executable should match folder's name of the demo.

Reply:
@Wovchena done